### DISCUSSION TITLE
EOReader and AWS

### DISCUSSION THREAD

**Discussion by bcerripromethee** [sertit/eoreader#102] [Q&A] [answered]
Hey,

Does EOReader works with online images, such as Sentinel-2 images provided by element84 (_https://earth-search.aws.element84.com/v1/search_)

I know that rioxarray is able to read the bands, one by one with the band link, such as _https://sentinel-cogs.s3.us-west-2.amazonaws.com/sentinel-s2-l2a-cogs/31/U/DQ/2023/7/S2A_31UDQ_20230714_0_L2A/B11.tif_

But you lost all of the structure of the .SAFE data.

Do you have any clue about that ?

Cheers,
Bastien

**Comment by remi-braun** [comment, marked as answer]
Hello,

That's a good question.
Sadly for now EOReader only works with SAFE structure (on disk or on the cloud) so this cannot work.
But this feature will certainly come in a next release 😉 
Stay tuned!

**Reply by remi-braun** [reply]
Issue created #104
